### ISSUE TITLE
promisify?

### ISSUE THREAD

**Issue by jamesdixon** [aexmachina/factory-girl#23]
At this point, you probably hate me for asking 20 questions, but last one, I promise :)

I saw that there was a `factory.promisify` function in the code where you can pass a promisify library into it. I tried this with Bluebird, but no luck -- I get an undefined error. Does this need to be called after all of the factories have been defined in order to work properly?

Thank you!
James

**Comment by aexmachina** [comment]
That's okay. It should work. Can you send the code you're using.

**Comment by ejlangev** [comment]
This is the code in the example doesn't work for me.  Here is the code I just implemented that is working (I'm using version 2.0.0):

```js
var promise = require('bluebird');
var FactoryGirl = require('factory-girl');
var Factory = new FactoryGirl.Factory().promisify(promise);

Factory.define(...);
```

**Comment by jamesdixon** [comment]
Had forgotten about this! Thanks @ejlangev - I'll give it a shot!
